SeniorSlideshowProject/src/SlideshowMaker.java

```diff
@@ -26,6 +26,8 @@
 public class SlideshowMaker extends JFrame {
 
 	private static final long serialVersionUID = 1L;
+	
+	private SlideShowStateMachine stateMachine;

```

Comment:
For some reason your PR has an outdated version of this file. There is already a state machine in the newest version. I'd try to re-update from master. Or maybe close this PR and open a new one if you already have the laster version of the file in there. I'm hesitant to say you should commit it if it overwrites the work already done.